client/subscription-edit-page.js

```diff
@@ -0,0 +1,28 @@
+/* global wcpaySubscriptionEdit */
+
+const addWCPayCards = ( { gateway, table, metaKey, tokens } ) => {
+	const paymentMethodInputId = `_payment_method_meta[${ gateway }][${ table }][${ metaKey }]`;
+	const paymentMethodInput = document.getElementById( paymentMethodInputId );
+
+	// Abort if the input doesn't exist or is already a select element
+	if ( ! paymentMethodInput || 'SELECT' === paymentMethodInput.tagName ) {
+		return;
+	}
+
+	const paymentMethodSelect = document.createElement( 'select' );
+	tokens.forEach( ( token ) => {
+		const option = document.createElement( 'option' );
+		option.value = token.tokenId;
+		option.text = token.displayName;
+		paymentMethodSelect.appendChild( option );
+	} );
+	paymentMethodSelect.id = paymentMethodInputId;
+	paymentMethodSelect.name = paymentMethodInputId;
+	paymentMethodSelect.value = paymentMethodInput.value;
```

Comment:
`value` can be the empty string, so in that case it would be good to add an `<option value="">Choose one...</option>` element as the first option so it's clear that no payment method is selected.

Reply:
Good idea! Added in https://github.com/Automattic/woocommerce-payments/pull/892/commits/da3a59f2832fb9967a9a770395d217f6e32602bd.